### ISSUE TITLE
Manual sample entry doesn't get validated

### ISSUE THREAD

**Issue by mitalia** [chop-dbhi/varify#224]
If a user manually modifies their sample to some ID that is invalid, the whole UI goes haywire

![image](https://cloud.githubusercontent.com/assets/21576/2618385/304c8bd6-bc20-11e3-9360-ca2a714160a7.png)

**Comment by naegelyd** [comment]
When entering a bad value, the `api/contexts` endpoint returns `422 -- Unprocessable Entry`  with the following error:

``` json
{
    "errors": {
        "json": [
            "\"XYZ\" is not a valid value for a primary key."
        ]
    },
    "message": "Error creating context"
}
```

**Comment by naegelyd** [comment]
Relies on a new release of Cilantro containing fixes for cbmi/cilantro#513 and cbmi/cilantro#514.

**Comment by naegelyd** [comment]
Fixed by upgrade to Cilantro `2.2.25` in ea0fcf4f7726d20a2802c3f4.
